openassessment/xblock/test/test_studio.py

```diff
@@ -304,6 +300,7 @@ def _assert_rendered_editor_order(self, xblock, expected_assessment_order):
         actual_assessment_order = [
             index_dict['name']
             for index_dict in sorted(assessment_indices, key=lambda d: d['index'])
+            if index_dict['index'] > 0
```

Comment:
Turns out, `find()` returns -1 if a value is not found, which makes that value always appear at the front of `actual_assessment_order` even if it is not present at all.